examples/cosmos/lightclient/Blockchain.qnt

```diff
@@ -154,14 +154,11 @@ module Blockchain {
 
     // (********************* BLOCKCHAIN ACTIONS ********************************)
     // Advance the clock by zero or more time units.
```

Comment:
Fair enough, this is actually one of the operators I had to change, originally it was nullary and internally picked tm nondeterministically, but I made it take a parameter, so that it worked with the instance framework of Quint.